<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <comment>
    * Released as open source by Gabriel Caudrelier
    *
    * Developed by Gabriel Caudrelier, gabriel dot caudrelier at gmail dot com
    *
    * https://github.com/metrodango/pip3line
    *
    * Released under BSD 3-Clause see LICENSE for more information
  </comment>
 <class>ModulesManagementWidget</class>
 <widget class="QWidget" name="ModulesManagementWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="infoLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="pathLabel">
     <property name="text">
      <string>Looking for modules in paths: </string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="pathListWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="loadedLabel">
     <property name="text">
      <string>Currently loaded modules:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="modulesListWidget">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="loadPushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Load module</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
